### DISCUSSION TITLE
Export cameras parameters as txt or xmp

### DISCUSSION THREAD

**Discussion by Mxicklejx** [alicevision/Meshroom#1886] [Q&A]
Hello, I really need to export cameras from meshroom to another photogrammetry software sadly i dont have any knowledge in this.. It seems like i have two import options there either as txt (XYZ, XZY, opengl) in two files (internals and externals) or .xmp workflow.. If its possible to export these types of files from meshroom any help would be appreciated.

**Comment by natowi** [comment]
You can use https://meshroom-manual.readthedocs.io/en/latest/feature-documentation/nodes/ConvertSfMFormat.html
to convert the sfm file to abc’, ‘sfm’, ‘json’, ‘ply’ or ‘baf. 
Only select Export views to get the camera positions only.
Meshlab or other tools may be of help to export ply or abc to another format
